source/fem-dem/parameters_cfd_dem.cc

```diff
@@ -223,6 +223,30 @@ namespace Parameters
         "Invalid vans model. Valid choices are modelA and modelB."));
     prm.leave_subsection();
   }
+
+  void
+  CFDDEM_postprocessing::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("post-processing");
+    prm.declare_entry("calculate total volume",
```

Comment:
I feel like total volume does not give you an idea of what volume you are referring to.